components/google_sheets/sources/common.js

```diff
@@ -16,6 +16,14 @@ module.exports = {
       },
     },
     watchedDrive: { propDefinition: [google_sheets, "watchedDrive"] },
+    initialEventCount: {
```

Comment:
We actually don't need to make this a prop or make it optional. You can simply emit the events (assuming the user has records in the specified worksheet). With Zapier and other platforms, this is the default, too, so the user doesn't have to think about it - they just get data.

Reply:
I transformed it to a constant instead.